Mathlib/MeasureTheory/Group/Measure.lean

```diff
@@ -246,6 +248,27 @@ end MeasurableMul
 
 end Mul
 
+section Semigroup
+
+variable [Semigroup G] [MeasurableMul G]
+
+/-- The image of a left invariant measure under scalar multiplication is left invariant. -/
```

Comment:
Although we use `SMul`, the name "scalar multiplication is really misleading here". 
```suggestion
/-- The image of a left invariant measure under a left action is left invariant, assuming 
that the action preserves multiplication. -/
```
Similar for the additive version and all other docstrings.